ceph-iscsi.spec

```diff
@@ -133,6 +133,13 @@ mkdir -p %{buildroot}%{_mandir}/man8
 install -m 0644 gwcli.8 %{buildroot}%{_mandir}/man8/
 gzip %{buildroot}%{_mandir}/man8/gwcli.8
 mkdir -p %{buildroot}%{_unitdir}/rbd-target-gw.service.d
+# When rtslib is fixed drop  this and update the dep. Note that we add both
+# etc and var, because the kernel and userspace developers keep switching
```

Comment:
```suggestion
# When rtslib is fixed drop this and update the dep. Note that we add both
# /etc and /var, because the kernel and userspace developers keep switching
```

Reply:
Sure, I just copied it from downstream. will fix it.